### ISSUE TITLE
Feature Request: When enabling only 1 OAuth login methods, redirect automatically

### ISSUE THREAD

**Issue by andrewcole** [eikek/sharry#780]
As an enhancement to #307, would it be possible to start the OAuth login flow automatically if there is only 1 OAuth method enabled?

Should be as simple as redirecting to /api/v2/open/auth/oauth/<ID> from /app/login, but I'm not familiar with scala at all so I may be underestimating it!

**Comment by andrewcole** [comment]
I tried this out by making my forwarding proxy redirect requests to /app/login to /api/v2/open/auth/oauth/XXX, and I had to exclude any requests with ?oauth=1, but otherwise it works well. I used the following in my Caddyfile:

```
@login {
  path /app/login
  method GET
  not query oauth=1
}
redir @login /api/v2/open/auth/oauth/XXX
```

Does make the "logout" option in the menu pointless though, as it logs the user out, then redirects them to the login flow, and logs them back in again! Maybe that option should be hidden as well?
